packages/js/product-editor/src/components/header/plugin-header-items/types.tsx

```diff
@@ -0,0 +1,12 @@
+export type PluginHeaderItemModalProps = {
+	children?: React.ReactNode;
+	icon: React.ReactNode;
+	label?: string;
+	title: string;
+};
+
+export type PluginHeaderItemPopoverProps = {
+	children?: React.ReactNode;
+	icon: React.ReactNode;
```

Comment:
`icon` should be optional, and default to a generic plugin icon if not provided.